src/main/webapp/index.html

```diff
@@ -29,7 +29,7 @@
             <!--TODO: Put in links to other pages. Put on all pages-->
             <a class="nav-item nav-link" href="#">Profile</a>
             <a class="nav-item nav-link" href="#">Events</a>
-            <a class="nav-item nav-link" href="#">Availability</a>
+            <a class="nav-item nav-link" href="availability.html">Availability</a>
```

Comment:
How are we doing this navbar incorporation? l thought we would leave all the links as "#" for now and then create a new branch after all the HTMLs are added into master, then add all links to all of their navbars. lf we're sticking with this way, can you remove the `href="availability.html"` here for consistency? lt bugs me how we have  `href="availability.html"` but not the same for Profile.

Reply:
Now that I pulled from master, I updated the navbars to the current state of this branch (links for all but scheduling and logout).